android/build.gradle

```
@@ -55,6 +55,6 @@ android {
 
 dependencies {
     implementation("org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version")
-    implementation("io.didomi.sdk:android:2.4.0")
+    implementation("io.didomi.sdk:android:2.5.1")
```

You should use the very last version
```suggestion
    implementation("io.didomi.sdk:android:2.6.0")
```